README.md

```diff
@@ -74,7 +86,7 @@ We have the following branches
  * **gh-pages**
    This contains the autogenerated code of the master branch that is generated by Travis.
  * **generator**
-   This branch is responsible for having the changes which will be implemented for generating the iso using webapp [meilix-generator](https://github.com/fossasia/meilix-generator)
+   This branch is responsible for having the changes which will be implemented for generating the iso using webapp [meilix-generator](https://github.com/fossasia/meilix-generator). It basically fetch the latest release from Github and use [mksquashfs tool](https://github.com/fossasia/meilix/blob/generator/build1.sh) to extract, made changes and then repack it and mail it. This will take very less time to customize the ISO.
```

Comment:
He she it -> s at the end of the verb "it fetches, extracts, repacks, mails". 

less is a comparism. maybe "not much" and if you havea time, you can say about x minutes.

Reply:
@xeon-zolt /
Can you help me to mention the exact time that gets decreased, please?

Reply:
10-20 minutes